src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -661,6 +662,7 @@ private void AddTypeKind(INamedTypeSymbol symbol)
                 {
                     switch (symbol.TypeKind)
                     {
+                        // FindValidCloneMethod is called instead of symbol.IsRecord since we want records in VB compilations to display as records.
```

Comment:
>// FindValidCloneMethod is called instead of symbol.IsRecord since we want records in VB compilations to display as records. [](start = 24, length = 124)

I think symbol display should work based on public API, this is how it is designed to work. Based on IsRecord value in this case.

Reply:
If we want this behavior with VB symbols, then IsRecord implementation for VB should be adjusted to recognize C# records

---
In reply to: [501405673](https://github.com/dotnet/roslyn/pull/47960#discussion_r501405673) [](ancestors = 501405673)